cmd/main.go

```diff
@@ -160,3 +181,25 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// getWatchNamespace returns the Namespace the operator should be watching for changes
+func getWatchNamespace() (string, error) {
+	var watchNamespaceEnvVar = "WATCH_NAMESPACE"
```

Comment:
can you please change for WATCH_NAMESPACE in makefile and container ENV as we did for serviceaccount_name_prefix https://github.com/ceph/ceph-csi-operator/pull/116?

Reply:
I'm trying not to touch manifests as during backports it's always creating extra work and that should be fixed first.

Reply:
please open a tracker if you are planning to fix it as followupl